test/libsolidity/syntaxTests/inheritance/override/triangle_no_impl.sol

```diff
@@ -0,0 +1,5 @@
+abstract contract A { function f() public pure virtual; }
+contract B is A { function f() public pure virtual override {} }
+contract C is A, B { }
```

Comment:
Is this fine because one of the bases doesn't provide an implementation?

Reply:
And because ``A`` is a common base, so it's the function is "intentionally" the same...